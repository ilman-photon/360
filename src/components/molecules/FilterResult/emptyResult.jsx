import { Box } from "@mui/system";
import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
import { Typography } from "@mui/material";

export const EmptyResult = ({ message = "" }) => {
  return (
    <Box sx={{ display: "flex", flexDirection: "column" }}>
      <CalendarTodayIcon
        sx={{ alignSelf: "center", width: 38.5, height: 38.5 }}
      />
      <Typography
        variant="bodyLarge"
        sx={{ fontWeight: 500, color: "#191919", textAlign: "center" }}
      >
        {message}
      </Typography>
    </Box>
  );
};

export default EmptyResult;
