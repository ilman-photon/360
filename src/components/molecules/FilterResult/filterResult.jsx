import { Box } from "@mui/system";
import constants from "../../../utils/constants";
import ItemResult from "../../organisms/ItemResult/ItemResult";
import FilterResultHeading from "../FilterResultHeading/filterResultHeading";

export const FilterResult = ({
  dataCount = 3,
  onSelectSchedule = () => {
    // This is intentional
  },
}) => {
  function renderItemResult() {
    const indents = [];
    for (let i = 0; i < dataCount; i++) {
      indents.push(<ItemResult />);
    }
    return indents;
  }

  return (
    <>
      <Box>
        <Box>
          <FilterResultHeading />
        </Box>
        {renderItemResult()}
      </Box>
    </>
  );
};

export default FilterResult;
