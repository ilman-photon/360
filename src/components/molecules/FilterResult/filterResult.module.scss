// TODO : move style to this file
.stackListContainer {
  .prevArrowActive {
    cursor: pointer;
    opacity: 1;
  }
  .prevArrowDisable {
    opacity: 0.3;
    cursor: not-allowed;
  }
}
