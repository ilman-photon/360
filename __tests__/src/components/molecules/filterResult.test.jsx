import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import FilterResult from "../../../../src/components/molecules/FilterResult/filterResult";

window.scrollTo = jest.fn()

describe("FilterResult Components", () => {
  let container;
  beforeEach(() => {
    container = render(<FilterResult/>);
  });

  it("FilterResult render", () => {
    expect(container).toMatchSnapshot();
  });

  it("FilterHeading render", () => {
    container = render(<FilterResult isDesktop={true}/>);
    expect(container).toMatchSnapshot();
  });

});
