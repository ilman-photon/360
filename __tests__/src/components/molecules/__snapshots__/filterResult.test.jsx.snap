// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterResult Components FilterHeading render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-nen11g-MuiStack-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="mobileFilterContainer MuiBox-root css-0"
          >
            <div
              class="css-19ki0tq-MuiStack-root"
            >
              <div
                class="mobileFilterStyle css-m69qwo-MuiStack-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-tqc5qm-MuiSvgIcon-root"
                  data-testid="LocationOnOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                  />
                  <circle
                    cx="12"
                    cy="9"
                    r="2.5"
                  />
                </svg>
                <div
                  class="mobileFilterTitleContainer MuiBox-root css-0"
                >
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium mobileFilterTitle css-37jdci-MuiTypography-root"
                  >
                    New York, NY
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 mobileFilterSubtitle css-ahj2mt-MuiTypography-root"
                  >
                    Eye exam * Aethna
                  </p>
                </div>
                <div
                  class="mobileFilterImageContainer MuiBox-root css-0"
                >
                  <span
                    style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                  >
                    <span
                      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2731%27%20height=%2731%27/%3e"
                        style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                      />
                    </span>
                    <img
                      alt=""
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </span>
                </div>
              </div>
            </div>
            <div />
          </div>
        </div>
        <div
          class="hide-scrollbar"
          style="overflow: auto; margin-top: 8px; max-height: calc(100vh - 151px - 64px - 141px);"
        />
      </div>
    </div>
    <div>
      <div
        class="css-nen11g-MuiStack-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="filterContainer MuiBox-root css-0"
          >
            <div
              class="filterButtonContainer MuiBox-root css-0"
            >
              <button
                class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root filterButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
                data-testid="filterbtn"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium tuneIcon css-i4bv87-MuiSvgIcon-root"
                  data-testid="TuneIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z"
                  />
                </svg>
                Filter
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium keydownIcon css-i4bv87-MuiSvgIcon-root"
                  data-testid="KeyboardArrowDownIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.41 8.59 12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
            <div
              class="MuiBox-root css-sesgiw"
            >
              <div
                class="filterFoundWarpper MuiBox-root css-jy5pi"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 filterFound css-ahj2mt-MuiTypography-root"
                >
                  30 In-network providers
                </p>
              </div>
              <div
                class="MuiBox-root css-1qe922y"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w23uea-MuiSvgIcon-root"
                  data-testid="ArrowBackIosIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11.67 3.87 9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"
                  />
                </svg>
                <div
                  class="MuiBox-root css-g9lo0d"
                >
                  <div
                    class="calenderDayWarpper MuiBox-root css-1prvghn"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      mon
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 19
                    </p>
                  </div>
                  <div
                    class="calenderDayWarpper MuiBox-root css-19wmdl0"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      tue
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 20
                    </p>
                  </div>
                  <div
                    class="calenderDayWarpper MuiBox-root css-114qxfe"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      wed
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 21
                    </p>
                  </div>
                  <div
                    class="calenderDayWarpper MuiBox-root css-cg2h46"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      thu
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 22
                    </p>
                  </div>
                  <div
                    class="calenderDayWarpper MuiBox-root css-oacha6"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      fri
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 23
                    </p>
                  </div>
                  <div
                    class="calenderDayWarpper MuiBox-root css-1ibu2l3"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                    >
                      sat
                    </p>
                    <p
                      class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                    >
                      Sep 24
                    </p>
                  </div>
                </div>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-x1a2ui-MuiSvgIcon-root"
                  data-testid="ArrowForwardIosIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6.23 20.23 8 22l10-10L8 2 6.23 3.77 14.46 12z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <div
          class="hide-scrollbar"
          style="overflow: auto; margin-top: 8px; max-height: calc(100vh - 151px - 64px - 141px);"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-nen11g-MuiStack-root"
    >
      <div
        class="MuiBox-root css-0"
      >
        <div
          class="filterContainer MuiBox-root css-0"
        >
          <div
            class="filterButtonContainer MuiBox-root css-0"
          >
            <button
              class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root filterButton css-19s7hx2-MuiButtonBase-root-MuiButton-root"
              data-testid="filterbtn"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium tuneIcon css-i4bv87-MuiSvgIcon-root"
                data-testid="TuneIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z"
                />
              </svg>
              Filter
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium keydownIcon css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.41 8.59 12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiBox-root css-sesgiw"
          >
            <div
              class="filterFoundWarpper MuiBox-root css-jy5pi"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 filterFound css-ahj2mt-MuiTypography-root"
              >
                30 In-network providers
              </p>
            </div>
            <div
              class="MuiBox-root css-1qe922y"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w23uea-MuiSvgIcon-root"
                data-testid="ArrowBackIosIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.67 3.87 9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"
                />
              </svg>
              <div
                class="MuiBox-root css-g9lo0d"
              >
                <div
                  class="calenderDayWarpper MuiBox-root css-1prvghn"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    mon
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 19
                  </p>
                </div>
                <div
                  class="calenderDayWarpper MuiBox-root css-19wmdl0"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    tue
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 20
                  </p>
                </div>
                <div
                  class="calenderDayWarpper MuiBox-root css-114qxfe"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    wed
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 21
                  </p>
                </div>
                <div
                  class="calenderDayWarpper MuiBox-root css-cg2h46"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    thu
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 22
                  </p>
                </div>
                <div
                  class="calenderDayWarpper MuiBox-root css-oacha6"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    fri
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 23
                  </p>
                </div>
                <div
                  class="calenderDayWarpper MuiBox-root css-1ibu2l3"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay css-ahj2mt-MuiTypography-root"
                  >
                    sat
                  </p>
                  <p
                    class="MuiTypography-root MuiTypography-body1 calenderDay calenderMonth css-ahj2mt-MuiTypography-root"
                  >
                    Sep 24
                  </p>
                </div>
              </div>
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-x1a2ui-MuiSvgIcon-root"
                data-testid="ArrowForwardIosIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M6.23 20.23 8 22l10-10L8 2 6.23 3.77 14.46 12z"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
      <div
        class="hide-scrollbar"
        style="overflow: auto; margin-top: 8px; max-height: calc(100vh - 151px - 64px - 141px);"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`FilterResult Components FilterResult render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-nen11g-MuiStack-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="mobileFilterContainer MuiBox-root css-0"
          >
            <div
              class="css-19ki0tq-MuiStack-root"
            >
              <div
                class="mobileFilterStyle css-m69qwo-MuiStack-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-tqc5qm-MuiSvgIcon-root"
                  data-testid="LocationOnOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                  />
                  <circle
                    cx="12"
                    cy="9"
                    r="2.5"
                  />
                </svg>
                <div
                  class="mobileFilterTitleContainer MuiBox-root css-0"
                >
                  <span
                    class="MuiTypography-root MuiTypography-bodyMedium mobileFilterTitle css-37jdci-MuiTypography-root"
                  >
                    New York, NY
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 mobileFilterSubtitle css-ahj2mt-MuiTypography-root"
                  >
                    Eye exam * Aethna
                  </p>
                </div>
                <div
                  class="mobileFilterImageContainer MuiBox-root css-0"
                >
                  <span
                    style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                  >
                    <span
                      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2731%27%20height=%2731%27/%3e"
                        style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                      />
                    </span>
                    <img
                      alt=""
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </span>
                </div>
              </div>
            </div>
            <div />
          </div>
        </div>
        <div
          class="hide-scrollbar"
          style="overflow: auto; margin-top: 8px; max-height: calc(100vh - 151px - 64px - 141px);"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-nen11g-MuiStack-root"
    >
      <div
        class="MuiBox-root css-0"
      >
        <div
          class="mobileFilterContainer MuiBox-root css-0"
        >
          <div
            class="css-19ki0tq-MuiStack-root"
          >
            <div
              class="mobileFilterStyle css-m69qwo-MuiStack-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-tqc5qm-MuiSvgIcon-root"
                data-testid="LocationOnOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                />
                <circle
                  cx="12"
                  cy="9"
                  r="2.5"
                />
              </svg>
              <div
                class="mobileFilterTitleContainer MuiBox-root css-0"
              >
                <span
                  class="MuiTypography-root MuiTypography-bodyMedium mobileFilterTitle css-37jdci-MuiTypography-root"
                >
                  New York, NY
                </span>
                <p
                  class="MuiTypography-root MuiTypography-body1 mobileFilterSubtitle css-ahj2mt-MuiTypography-root"
                >
                  Eye exam * Aethna
                </p>
              </div>
              <div
                class="mobileFilterImageContainer MuiBox-root css-0"
              >
                <span
                  style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                >
                  <span
                    style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2731%27%20height=%2731%27/%3e"
                      style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                    />
                  </span>
                  <img
                    alt=""
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </span>
              </div>
            </div>
          </div>
          <div />
        </div>
      </div>
      <div
        class="hide-scrollbar"
        style="overflow: auto; margin-top: 8px; max-height: calc(100vh - 151px - 64px - 141px);"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
